app/livechat/server/startup.js

```diff
@@ -43,4 +46,6 @@ Meteor.startup(() => {
 			lng: user.language || settings.get('Language') || 'en',
 		}));
 	}, callbacks.priority.LOW, 'cant-leave-room');
+
+	createLivechatQueueView();
```

Comment:
this is the first time we use views, I think the best place to create them will be within a model.. maybe we should have a special model class for views